assets/boilerplate/theme.json

```diff
@@ -0,0 +1,303 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/theme.json",
+    "settings": {
+        "appearanceTools": true,
+        "border": {
+            "color": true,
+            "radius": true,
+            "style": true,
+            "width": true
+        },
+        "color": {
+            "custom": true,
+            "customGradient": true,
+            "link": true,
+            "palette": [
+                {
+                    "color": "#776c4e",
+                    "name": "Primary",
+                    "slug": "primary"
+                },
+                {
+                    "color": "#a19982",
+                    "name": "Secondary",
+                    "slug": "secondary"
+                },
+                {
+                    "color": "#000000",
+                    "name": "Foreground",
+                    "slug": "foreground"
+                },
+                {
+                    "color": "#ffffff",
+                    "name": "Background",
+                    "slug": "background"
+                },
+                {
+                    "color": "#f1ede6",
+                    "name": "Tertiary",
+                    "slug": "tertiary"
+                }
+            ]
+        },
+        "custom": { 
+            "gap": {
+                "horizontal": "min(30px, 5vw)",
+                "vertical": "min(30px, 5vw)"
+            }
+        },
+        "layout": {
+            "contentSize": "620px",
+            "wideSize": "1000px"
+        },
+        "spacing": {
+            "blockGap": true,
+            "margin": true,
+            "padding": true,
+            "units": [
+                "%",
+                "px",
+                "em",
+                "rem",
+                "vh",
+                "vw"
+            ]
+        },
+        "typography": {
+            "customFontSize": true,
+            "fontFamilies": [
+                {
+                    "fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+                    "name": "System Font",
+                    "slug": "system-font"
+                }
+           ],
+            "fontSizes": [
+                {
+                    "name": "Small",
+                    "size": "1rem",
+                    "slug": "small"
+                },
+                {
+                    "name": "Medium",
+                    "size": "1.5rem",
+                    "slug": "medium"
+                },
+                {
+                    "name": "Large",
+                    "size": "1.75rem",
+                    "slug": "large"
+                },
+                {
+                    "name": "Extra Large",
+                    "size": "2rem",
+                    "slug": "x-large"
+                }
+            ],
+            "lineHeight": true
```

Comment:
This lineHeight can be removed because appearance tools is already enabled :)
https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/#appearancetools